### PR TITLE
Fixes #1063. Un-break image uploads for issue comments.

### LINKED ISSUE #1063: [bug] Upload images is broken
Try to upload an image in the comments of a bug. For example

``` http
POST /upload/ HTTP/1.1
Host: webcompat.com
User-Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10.11; rv:48.0) Gecko/20100101 Firefox/48.0
Accept: */*
Accept-Language: en,fr;q=0.7,en-US;q=0.3
Accept-Encoding: gzip, deflate, br
DNT: 1
Referer: https://webcompat.com/issues/1930
x-requested-with: XMLHttpRequest
Content-Length: 63126
Content-Type: multipart/form-data; boundary=---------------------------1931572924435841969129253066
Cookie: session=.…cut…
Connection: keep-alive
```

And receiving from the server.

```http
HTTP/1.1 501 NOT IMPLEMENTED
Server: nginx/1.1.19
Date: Tue, 24 May 2016 06:04:49 GMT
Content-Type: text/html
Content-Length: 188
Connection: keep-alive
Set-Cookie: session=… cut…
```

with a flash message.

Ping @miketaylr :)

### PR DESCRIPTION
**Pull request by miketaylr**
r? @karlcow

### CODE REVIEW

File: webcompat/templates/issue/issue-comment-submit.html [hunk 1]
```diff
@@ -13,11 +13,12 @@
               <label class="wc-UploadForm-label wc-UploadForm-label--uploadForm">Attach images by <span class="wc-UploadForm-label--link">selecting them</span></label>
               <input accept=".jpe,.jpg,.jpeg,.gif,.png,.bmp" class="wc-ButtonUpload js-buttonUpload" id="image" name="image" type="file">
             </div>
-          </div>
+            <img class="wc-Upload-Loader wc-Issue-Loader js-Upload-Loader"
+                 src="/img/upload-loader.svg" alt="Uploading image">
+          </label>
```
Comment (miketaylr) [inline webcompat/templates/issue/issue-comment-submit.html]: (yes, we were closing `<label>` with `</div>`. 🙈)

Comment (karlcow) [inline webcompat/templates/issue/issue-comment-submit.html]: There's a bigger? issue. Nested labels. :)
https://www.w3.org/TR/html-markup/label.html


-     The label element must not contain any nested label elements.
-     The label element may contain at most one descendant input element, button element, select element, or textarea element.
-     The for attribute of the label element must refer to a form control.
-     The interactive element label must not appear as a descendant of the a element.
-     The interactive element label must not appear as a descendant of the button element.

Comment (magsout) [inline webcompat/templates/issue/issue-comment-submit.html]: oops. good catch. I didn't know. Thanks for thie huge information...

Comment (miketaylr) [inline webcompat/templates/issue/issue-comment-submit.html]: shrugs, it works. :p

Comment (miketaylr) [inline webcompat/templates/issue/issue-comment-submit.html]: I'd recommend we *not* block image uploads on the nested labels issue and file a bug. Would you like to or want me to @karlcow?

Comment (miketaylr) [inline webcompat/templates/issue/issue-comment-submit.html]: Issue filed here: https://github.com/webcompat/webcompat.com/issues/1070

**Comment by miketaylr** [comment]
Any other feedback?

**Comment by karlcow** [comment]
ok. excellent. Let's move on. Thanks for filling the bug on labels. r+ @miketaylr

**Comment by miketaylr** [comment]
Thanks for review Karl!

**Pull request merged**